proposals/kroxylicious-operator-api-alternative.md

```diff
@@ -231,20 +231,8 @@ spec:
          x: y
 
   # Points to the cluster being proxied.  Can either be Strimzi Kafka resource or endpoint details.
-  targetCluster:
-    clusterRef:
-      group: strimzi.io|kroxylicious.io # must be strimzi.io or kroxylicious.io
-      kind: Kafka|KafkaClusterRef  # must be Kafka (belonging to group strimzi.io) or TargetClusterRef (belonging to group kroxylicious.io)
-      name: my-cluster
-      listenerName: listener # name of strimzi listener, permitted only for Kafka (group strimzi.io).  (Optional)
-    tls:
-      # Optional - client auth certificate
-      # secret provided by the Developer.
-      certificateRef:
```

Comment:
are we happy to have the client cert specified by the kafka service ref? If we have it on the virtual cluster then different virtual clusters can have different certificates for the same kafka service. Else the user would have to create two KafkaService's with the same details except for the client certs.

Or we could support some combination of the two, allowing VCluster to override the client certs